INFO 2023-09-25 17:38:01,253 autoreload 16604 2228 Watching for file changes with StatReloader
INFO 2023-09-25 17:38:03,416 basehttp 16604 7568 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 17:38:08,382 basehttp 16604 7568 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 17:38:12,671 basehttp 16604 7568 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 17:38:29,797 basehttp 16604 7568 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 17:42:08,716 autoreload 13880 2260 Watching for file changes with StatReloader
INFO 2023-09-25 17:42:10,973 basehttp 13880 6056 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 17:42:12,559 basehttp 13880 6056 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 18:52:04,164 autoreload 2904 4948 Watching for file changes with StatReloader
INFO 2023-09-25 18:52:05,960 basehttp 2904 12844 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 18:52:08,191 basehttp 2904 12844 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 18:52:09,456 basehttp 2904 12844 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 18:52:26,698 autoreload 2904 4948 E:\GB\Django\HW\HW\urls.py changed, reloading.
INFO 2023-09-25 18:52:26,952 autoreload 16852 12592 Watching for file changes with StatReloader
INFO 2023-09-25 18:53:00,579 autoreload 16852 12592 E:\GB\Django\HW\HW\urls.py changed, reloading.
INFO 2023-09-25 18:53:00,839 autoreload 13500 15100 Watching for file changes with StatReloader
INFO 2023-09-25 18:53:03,112 basehttp 13500 10716 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 18:53:05,409 basehttp 13500 10716 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 19:21:32,287 autoreload 11796 10144 Watching for file changes with StatReloader
INFO 2023-09-25 19:21:35,610 basehttp 11796 14184 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 19:21:38,287 basehttp 11796 14184 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 19:26:51,667 autoreload 6680 20204 Watching for file changes with StatReloader
INFO 2023-09-25 19:32:13,906 autoreload 20592 20760 Watching for file changes with StatReloader
INFO 2023-09-25 19:32:15,990 basehttp 20592 16816 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 19:32:17,780 basehttp 20592 16816 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 19:47:08,714 autoreload 19608 2264 Watching for file changes with StatReloader
INFO 2023-09-25 19:47:37,283 autoreload 19608 2264 E:\GB\Django\HW\hw\urls.py changed, reloading.
INFO 2023-09-25 19:47:37,561 autoreload 13528 16864 Watching for file changes with StatReloader
INFO 2023-09-25 19:47:38,692 basehttp 13528 15960 "GET / HTTP/1.1" 200 865
INFO 2023-09-25 19:47:40,582 basehttp 13528 15960 "GET /about/ HTTP/1.1" 200 666
INFO 2023-09-25 19:47:41,877 basehttp 13528 15960 "GET /about/ HTTP/1.1" 200 666
INFO 2023-10-01 18:01:40,292 autoreload 2948 17128 Watching for file changes with StatReloader
INFO 2023-10-01 18:05:37,441 autoreload 1740 8032 Watching for file changes with StatReloader
INFO 2023-10-01 18:06:13,163 autoreload 1740 8032 E:\GB\Django\HW\hw4\forms.py changed, reloading.
INFO 2023-10-01 18:06:13,435 autoreload 18836 14076 Watching for file changes with StatReloader
INFO 2023-10-01 18:06:15,862 autoreload 19584 2376 Watching for file changes with StatReloader
INFO 2023-10-01 18:06:59,727 autoreload 19584 2376 E:\GB\Django\HW\hw4\urls.py changed, reloading.
INFO 2023-10-01 18:07:00,009 autoreload 10160 8932 Watching for file changes with StatReloader
INFO 2023-10-01 18:08:23,344 autoreload 10160 8932 E:\GB\Django\HW\hw\settings.py changed, reloading.
INFO 2023-10-01 18:08:23,615 autoreload 1700 17668 Watching for file changes with StatReloader
INFO 2023-10-01 18:08:28,982 autoreload 20212 15016 Watching for file changes with StatReloader
INFO 2023-10-01 18:08:45,326 autoreload 20212 15016 E:\GB\Django\HW\hw4\apps.py changed, reloading.
INFO 2023-10-01 18:08:45,631 autoreload 9984 340 Watching for file changes with StatReloader
INFO 2023-10-01 18:10:27,445 autoreload 9984 340 E:\GB\Django\HW\hw2\views.py changed, reloading.
INFO 2023-10-01 18:10:27,710 autoreload 12956 16652 Watching for file changes with StatReloader
INFO 2023-10-01 18:11:15,676 autoreload 12956 16652 E:\GB\Django\HW\hw\urls.py changed, reloading.
INFO 2023-10-01 18:11:15,964 autoreload 17896 19272 Watching for file changes with StatReloader
WARNING 2023-10-01 18:11:18,054 log 17896 8432 Not Found: /
WARNING 2023-10-01 18:11:18,054 basehttp 17896 8432 "GET / HTTP/1.1" 404 2275
WARNING 2023-10-01 18:11:18,155 log 17896 8432 Not Found: /favicon.ico
WARNING 2023-10-01 18:11:18,156 basehttp 17896 8432 "GET /favicon.ico HTTP/1.1" 404 2326
ERROR 2023-10-01 18:11:58,932 basehttp 17896 17276 "GET /homework3/ HTTP/1.1" 500 59
INFO 2023-10-01 18:13:41,838 autoreload 17896 19272 E:\GB\Django\HW\hw\urls.py changed, reloading.
INFO 2023-10-01 18:13:42,134 autoreload 17704 9004 Watching for file changes with StatReloader
INFO 2023-10-01 18:14:38,797 autoreload 15124 12396 Watching for file changes with StatReloader
INFO 2023-10-01 18:14:40,550 basehttp 15124 12552 "GET / HTTP/1.1" 200 865
INFO 2023-10-01 18:14:43,737 basehttp 15124 12552 "GET /about/ HTTP/1.1" 200 666
ERROR 2023-10-01 18:15:08,427 basehttp 15124 12552 "GET /homework3/ HTTP/1.1" 500 59
INFO 2023-10-01 18:16:15,702 autoreload 15124 12396 E:\GB\Django\HW\hw4\urls.py changed, reloading.
INFO 2023-10-01 18:16:15,997 autoreload 18648 10480 Watching for file changes with StatReloader
INFO 2023-10-01 18:16:17,815 basehttp 18648 2988 "GET / HTTP/1.1" 200 865
INFO 2023-10-01 18:16:21,523 basehttp 18648 2988 "GET /about/ HTTP/1.1" 200 666
WARNING 2023-10-01 18:16:41,485 log 18648 2988 Not Found: /index
WARNING 2023-10-01 18:16:41,485 basehttp 18648 2988 "GET /index HTTP/1.1" 404 2636
INFO 2023-10-01 18:16:53,368 basehttp 18648 2988 "GET /homework3 HTTP/1.1" 301 0
ERROR 2023-10-01 18:16:53,752 basehttp 18648 16832 "GET /homework3/ HTTP/1.1" 500 59
INFO 2023-10-01 18:18:36,593 autoreload 9164 13876 Watching for file changes with StatReloader
ERROR 2023-10-01 18:18:45,916 basehttp 9164 15444 "GET /homework3/ HTTP/1.1" 500 59
INFO 2023-10-01 18:19:25,658 autoreload 9164 13876 E:\GB\Django\HW\hw\settings.py changed, reloading.
INFO 2023-10-01 18:19:25,941 autoreload 7724 11588 Watching for file changes with StatReloader
INFO 2023-10-01 18:19:27,157 basehttp 7724 17728 "GET / HTTP/1.1" 200 865
ERROR 2023-10-01 18:19:29,979 log 7724 17728 Internal Server Error: /homework3/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\engine.py", line 161, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
django.template.exceptions.TemplateDoesNotExist: myapp2/base.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw2\views.py", line 8, in index
    return render(request, "hw2/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\backends\django.py", line 63, in render
    reraise(exc, self.backend)
  File "E:\GB\Django\venv\Lib\site-packages\django\template\backends\django.py", line 84, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: myapp2/base.html
ERROR 2023-10-01 18:19:29,981 basehttp 7724 17728 "GET /homework3/ HTTP/1.1" 500 131607
INFO 2023-10-01 18:20:17,906 autoreload 19264 10056 Watching for file changes with StatReloader
INFO 2023-10-01 18:20:19,998 basehttp 19264 17640 "GET /homework3/ HTTP/1.1" 200 1075
ERROR 2023-10-01 18:21:46,932 log 19264 17640 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:21:46,934 basehttp 19264 17640 "GET /homework4/ HTTP/1.1" 500 80798
INFO 2023-10-01 18:23:18,913 autoreload 15832 18760 Watching for file changes with StatReloader
ERROR 2023-10-01 18:23:24,291 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:24,292 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:24,994 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:24,995 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:25,243 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:25,244 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:25,446 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:25,448 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:25,667 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:25,668 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:26,507 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:26,508 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:26,791 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:26,792 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:27,013 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:27,014 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
ERROR 2023-10-01 18:23:27,558 log 15832 17900 Internal Server Error: /homework4/
Traceback (most recent call last):
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\hw\hw4\views.py", line 58, in index
    return render(request, "hw4/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\GB\Django\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: hw4/index.html
ERROR 2023-10-01 18:23:27,560 basehttp 15832 17900 "GET /homework4/ HTTP/1.1" 500 80935
INFO 2023-10-01 18:30:34,617 autoreload 3448 16120 Watching for file changes with StatReloader
INFO 2023-10-01 18:30:34,841 basehttp 3448 16492 "GET /homework3/ HTTP/1.1" 200 1075
